payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -676,17 +639,9 @@ class LpmRepository constructor(
         }
     }
 
-    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-    sealed class ServerSpecState(val serverLpmSpecs: String?) {
```

Comment:
Never really used for much. It’s being replaced by a boolean return value for `update()`.